R/doc.R

```diff
@@ -197,16 +193,16 @@ extract.posfun <- function # Extract Positioning Method for documentation
   ## otherwise if one function's name is a substring of another's!
   ##L <- L[order(nchar(names(L)),decreasing=TRUE)]
   L
-### List of lists, each of which describes one Positioning Method
-### defined in f.
+  ### List of lists, each of which describes one Positioning Method
+  ### defined in f.
```

Comment:
@Anirban166 
can you please remove all spaces before triple # comments?
so the style is consistent throughout the package R code.
(that is the default when indenting in ESS)

Reply:
Sure! I don't remember adding them though, dunno where they came from, I also see other white space additions in 'Files changed' now that I am sure I did not make..are these added automatically?

Reply:
I am aware of the line endings that Git follow, but this is new, just noticed

Reply:
your editor may change white space when you do "auto-indent" and that can usually be configured (there should be some way to tell your editor to not put any spaces in front of triple # comments).
before doing "git commit" you should check the "git diff" to make sure there are only relevant changes that you want to show up on the diff to be code reviewed. If there are any unplanned changes (like white space here) then please remove them prior to doing "git commit"

Reply:
Sure, I have removed the unwanted space before all the ###, but lemme remove the other extra whitespaces as well

Reply:
Thanks for the information, I will try to look that up and configure it to not make unwanted changes!